example/java/prepare.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+cd $(dirname $0)
+
+# generate iroha lib
+if [[ $1 -eq 0 && ${IROHA_HOME} -eq 0 ]] ; then
```

Comment:
"$1", "${IROHA_HOME}"

in what case IROHA_HOME will be equal to 0? 

for emptiness better use -z

```
if [ -z "$1" && -z "${IROHA_HOME}" ]; then
```